Sources/Basic/JSON.swift

```diff
@@ -92,18 +92,18 @@ extension JSON: Equatable {
 
 extension JSON {
     /// Encode a JSON item into a string of bytes.
-    public func toBytes(prettyPrint: Bool = false) -> ByteString {
+    public func toBytes(prettyPrint: Bool = false, contentsKeyOrder: (String, String) -> Bool = {a, b in a < b}) -> ByteString {
```

Comment:
The closure can be written as `{ $0 < $1 }`. Following Swift conventions, the argument label should be named something like `sortingKeysBy` and the internal parameter should be`areInIncreasingOrder`.